Tests/LibPDF/TestPDF.cpp

```diff
@@ -267,3 +269,34 @@ TEST_CASE(postscript)
     check_evaluate("{ 3 1 roll }"sv, { 0.5f, 1.0f, 2.0f }, { 2.0f, 0.5f, 1.0f });
     check_evaluate("{ 3 -1 roll }"sv, { 0.5f, 1.0f, 2.0f }, { 1.0f, 2.0f, 0.5f });
 }
+
+TEST_CASE(render)
+{
+    auto file = MUST(Core::MappedFile::map("colorspaces.pdf"sv));
+    auto document = MUST(PDF::Document::create(file->bytes()));
+    MUST(document->initialize());
+    EXPECT_EQ(document->get_page_count(), 1U);
+
+    auto page = MUST(document->get_page(0));
+    auto page_size = Gfx::IntSize { 310, 370 };
+    auto bitmap = MUST(Gfx::Bitmap::create(Gfx::BitmapFormat::BGRx8888, page_size));
+    MUST(PDF::Renderer::render(document, page, bitmap, Color::White, PDF::RenderingPreferences {}));
```

Comment:
Shouldn't we prefer `TRY_OR_FAIL` over `MUST` in tests?

Reply:
I don't prefer it.